src/main/java/seedu/taassist/model/Model.java

```diff
@@ -99,6 +100,12 @@ public interface Model {
      */
     void deleteModuleClass(ModuleClass moduleClass);
 
+    /**
+     * Deletes multiple classes.
+     * The classes must exist in TA-Assist.
+     */
+    void deleteModuleClasses(Collection<ModuleClass> moduleClasses);
+
```

Comment:
How about `deleteModuleClasses(ModuleClass... moduleClass)` instead of these two methods? Might have pros and cons.

Reply:
I think I'll leave it as it is for now, because otherwise, we will have to convert `Collection<ModuleClass>` to an array everytime we want to delete multiple classes since varargs only accept Arrays.